### PR TITLE
Cleanup the metanorma functional spec file

### PR DESCRIPTION
**Pull request by abunashir**
Overtime the `spec/metanorma_spec` grew a lot and it becoming hard to maintain and also understand what's going on that. So, this commit tries to clean it up a bit and also simplify we were doing lots of stuff there.

As a side node, it made me thing if we actually need to have this test file with this repository, given that this are already being
tested in the metanorma library. These are bit repetitive and form the CLI prospective it only delegates some messages, and those are already being tested in the acceptance tests, but I would leave it it like this for now I guess.

Closes: #72

### CODE REVIEW

File: spec/metanorma_spec.rb [hunk 1]
```diff
@@ -203,6 +138,64 @@ def ruby(x)
...
+    @source_doc_file ||= output_path.join("test.adoc")
+  end
+
+  def create_clean_test_files(content)
+    cleanup_test_files
+    File.write(source_doc_file, content, encoding: "UTF-8")
+  end
+
+  def output_path
+    @output_path ||= Metanorma::Cli.root_path.join("tmp", "acceptance")
```
Comment (ronaldtse) [inline spec/metanorma_spec.rb:186]: Any reason we are not using Ruby tmpdir?
https://ruby-doc.org/stdlib-2.5.1/libdoc/tmpdir/rdoc/Dir.html

**Review by ronaldtse — CHANGES_REQUESTED** [review]
LGTM and excellent work -- just one question about using tmpdir

File: spec/metanorma_spec.rb (hunk 1 above)
Comment (abunashir) [reply spec/metanorma_spec.rb:186]: Good catch, I was testing it out with with local tmp directory for predictable output but I will change it to use the tmp directory next.

**Comment by abunashir** [comment]
Merging it now :)

**Pull request merged**